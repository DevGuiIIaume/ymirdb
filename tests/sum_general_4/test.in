set a 1 2 3
set b a 1
set c a 2
set d b c
forward a
forward b
forward c
forward d
backward a
backward c
backward b
backward d
sum a
sum b
sum c
sum d
len a
len b
len c
len d
bye
