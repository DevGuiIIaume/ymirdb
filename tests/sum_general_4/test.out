> ok

> ok

> ok

> ok

> nil

> a

> a

> a, b, c

> b, c, d

> d

> d

> nil

> 6

> 7

> 8

> 15

> 3

> 4

> 4

> 8

> bye
